src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -378,6 +378,18 @@ doclet.help.search.refer=Refer to the {0} for a full description of search featu
 doclet.help.search.spec.url=https://docs.oracle.com/en/java/javase/{0}/docs/specs/javadoc/javadoc-search-spec.html
 # The title for the Javadoc Search Specification
 doclet.help.search.spec.title=Javadoc Search Specification
+doclet.help.releases.head=Release Details
+doclet.help.releases.body.specify.top-level=\
+    The details for each module, package, class or interface \
+    normally include the release in which the declaration was introduced.
+doclet.help.releases.body.specify.member=\
+    The details for each member of a class or interface normally \
+    include the release in which the member was introduced, in those situations \
+    where the member was added after the initial introduction of the \
+    enclosing class or interface.
```

Comment:
I think "normally" is confusing here, because a member being added in a later release is not necessary the "normal" case. How about reversing the sentence to make it immediately clear what we're talking about?

> In those situations where a member was added after the initial introduction of the enclosing class or interface, the detail of the member includes the release in which it was introduced.

Reply:
Yeah, I agree the use of `normally` here could be seen as ambiguous

Reply:
I still think that reversing the sentence as proposed in my comment above would make the intent clearer. I think the "normally" is ok in that order, so I re-added it:

"In those situations where a member was added after the initial introduction of the enclosing class or interface, the detail of the member normally includes the release in which it was introduced."